cfgov/jinja2/v1/_includes/molecules/info-unit-2.html

```diff
@@ -0,0 +1,136 @@
+{# ==========================================================================
+
+   Info Unit
+
+   ==========================================================================
+
+   Description:
+
+   Create an info-unit molecule when given:
+
+   value:                        Object defined from a StreamField block.
+
+   value.modifier:               A modifier CSS class used to alternate
+                                 the info unit from stacked to inline layouts.
+
+   value.image:                  (Optional) An object containing:
+
+   value.image.is_decorative:    Whether the image is a bg image or not.
+                                 Default is false.
+
+   value.image.is_square:        Whether the image is square or not.
+                                 Default is false.
+
+   value.image.url:              String for the image URL.
+
+   value.image.alt:              String for image alt text.
+
+   value.heading:                The heading for the info unit.
+
+   value.heading_level:          The heading level (H2, H3, or H4).
+
+   value.heading_icon:           An optional cf-icons icon code that
+                                 triggers the insertion of an icon before
+                                 the heading.
+
+   value.body:                   A string containing the content to display
+                                 below the heading (typically a paragraph).
+
+   value.links:                  A tuple of items to create a list of links,
+                                 containing:
+
+   value.links[i].url:           A string for the URL of the link.
+
+   value.links[i].text:          A string for the text of the link.
+
+   value.is_button:              Boolean for whether the link should be
+                                 rendered as a button. Default is false.
+
+   value.link_image_and_heading: Boolean for whether the image and heading
+                                 should be linked to the value.links[0].url.
+
+   ========================================================================== #}
+
+{%- macro info_unit(value, format, link_image_and_heading) -%}
+
+{# Confirm that not only is link_image_and_heading set, but that this
+   particular info unit has a link in its list. #}
+{%- set link_image_and_heading = link_image_and_heading
+                                 and value.links
+                                 and value.links[0]
+                                 and value.links[0].url %}
+
+<div class="m-info-unit
+            {{- ' m-info-unit__inline' if format == '25-75' else '' }}">
+
+{%- if value.image.upload %}
+    {%- set img = image(value.image.upload, 'original') %}
+    {% if link_image_and_heading %}
+        <a href="{{ value.links[0].url }}">
+    {% endif %}
+        {%- if value.image.alt %}
+            <img class="m-info-unit_image
```

Comment:
I'm interested in this idea, but worried that it would have to be really convoluted to work for all of our use cases. It seems like something best explored in a future PR.